src/octree/tests.rs

```diff
@@ -37,24 +39,26 @@ mod octree_serialization_tests {
 
     #[test]
     fn test_big_octree_serialize() {
-        let mut tree = Octree::<u32>::new(512).ok().unwrap();
+        let red: Albedo = 0xFF0000FF.into();
```

Comment:
It would be nice to have some basic colors, like RED,GREEN,BLUE etc as constants built in, although this is not a requirement to approve the PR